src/quest_manager/tests/test_views.py

```diff
@@ -1481,6 +1481,27 @@ def test_show_hidden(self):
         # and no button when already viewing hidden quests
         self.assertNotContains(response, 'Show Hidden Quests')
 
+    def test_available_quest_list_ordering(self):
+        """Parses for queryset used as context for "Available Quests" list and asserts equal to a 
+        manually ordered queryset.
+        """
+
+        # log in as teacher and access available quests view
+        self.client.force_login(self.test_teacher)
+        staff_response = self.client.get(reverse('quests:available'))
+
+        # log in as student and access available quests view
+        self.client.force_login(self.test_student)
+        student_response = self.client.get(reverse('quests:available'))
+
+        # get queryset from view context and order it as intended
+        for response in [staff_response, student_response]:
+            displayed_order = response.context['available_quests']
+            intended_order = displayed_order.order_by("sort_order", "-time_expired", "-date_expired", "name")
```

Comment:
A code comment explaining where these order_by choices are coming from.

Reply:
switched to reference the XPItem model's meta option value for ordering and added context through code comment